api-get-object.go

```diff
@@ -44,8 +46,10 @@ func (c Client) GetEncryptedObject(bucketName, objectName string, encryptMateria
 		return nil, err
 	}
 
-	encryptMaterials.SetupDecryptMode(encReader, st.Metadata)
+	// Start decrypting the object.
```

Comment:
Decryption doesn't start here actually, this is just a setup. Data are decrypted when Read() is called.